services/forward-interop/Dockerfile

```diff
@@ -47,7 +47,8 @@ ENV PORT=4000 \
     INTEROP_PROXY_HOST='interop-proxy' \
     INTEROP_PROXY_PORT='8000' \
     TELEMETRY_HOST='telemetry' \
-    TELEMETRY_PORT='5000'
+    TELEMETRY_PORT='5000' \
+    UPLOAD_INTERVAL='500'
```

Comment:
Since the old default value was 200ms, the default for this one should also be 200ms so it won't change the behavior of existing things. The default value in `service.js` will be overrided by this env var value, and so the default value in `service.js` can be removed.